.github/workflows/update_openapi.yaml

```diff
@@ -25,18 +25,19 @@ jobs:
 
       - uses: peter-evans/create-pull-request@v3
         with:
-          title: "chore(openapi): update OpenAPI specification file"
+          title: "chore(openapi): update ${{ github.event.client_payload.id }} OpenAPI document"
           token: "${{ secrets.CI_USER_TOKEN }}"
-          commit-message: "chore(openapi): update OpenAPI specification file"
+          commit-message: "chore(openapi): update ${{ github.event.client_payload.id }} OpenAPI document"
           author: "app-services-ci <app-services-ci@users.noreply.github.com>"
-          branch: chore/add-openapi
+          branch: chore/add-${{ github.event.client_payload.id }}-openapi
```

Comment:
That will keep the same brach for API and aggregate changes on top of each other.
It wil be better pattern to merge all changes unquiely just in case there is some problem, revert neeeded or we have breaking change

Reply:
Yeah, you are right. The reason I opted against that is because we could have a number of open PRs with the same OpenAPI spec, and could run into conflicts when merging. But you are right, that would prevent overwriting.